__tests__/workers/notifications/userGiftedPlusNotification.ts

```diff
@@ -0,0 +1,94 @@
+import createOrGetConnection from '../../../src/db';
+import { DataSource } from 'typeorm';
+import { ChangeObject } from '../../../src/types';
+import { usersFixture } from '../../fixture';
+import { SubscriptionCycles } from '../../../src/paddle';
+import { addYears } from 'date-fns';
+import { invokeNotificationWorker, saveFixtures } from '../../helpers';
+import { PLUS_MEMBER_SQUAD_ID } from '../../../src/workers/userUpdatedPlusSubscriptionSquad';
+import { SourceType, User, Source } from '../../../src/entity';
+import { NotificationGiftPlusContext } from '../../../src/notifications';
+import { NotificationType } from '../../../src/notifications/common';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+type ObjectType = Partial<User>;
+const base: ChangeObject<ObjectType> = {
+  ...usersFixture[0],
+  subscriptionFlags: JSON.stringify({}),
+  createdAt: new Date().getTime(),
+  updatedAt: new Date().getTime(),
+  flags: JSON.stringify({}),
+};
+const plusUser = {
+  ...base,
+  subscriptionFlags: JSON.stringify({ cycle: SubscriptionCycles.Yearly }),
+};
+const giftedPlusUser = {
+  ...base,
+  subscriptionFlags: JSON.stringify({
+    cycle: SubscriptionCycles.Yearly,
+    gifterId: 2,
+    giftExpirationDate: addYears(new Date(), 1),
+  }),
+};
+
+beforeEach(async () => {
+  jest.resetAllMocks();
+  await saveFixtures(con, User, usersFixture);
+  await saveFixtures(con, Source, [
+    {
+      id: PLUS_MEMBER_SQUAD_ID,
+      name: 'Plus Squad',
+      image: 'http://image.com/a',
+      handle: 'plus-squad-notify',
+      type: SourceType.Squad,
+    },
+  ]);
+});
+
+const WORKER_RELATIVE_PATH =
+  '../../../src/workers/notifications/userGiftedPlusNotification';
+
+describe('plus subscription gift', () => {
+  it('should early return for currently non plus user', async () => {
```

Comment:
@ilasw no need to back adjust it, but we always want to have `should be registered` test for each worker, this ensures worker is setup correctly. Keep it in mind for the future.